File: server/Album.Domain/Photo.cs
using Shared.Domain;

namespace Albums.Domain
{
    public class Photo : Entity
    {
        public string ImageBase64 { get; set; }
        public string Description { get; set; }
        public string Title { get; set; }

        public Photo(string imageBase64, string description, string title)
        {
            ImageBase64 = imageBase64;
            Description = description;
            Title = title;
        }
    }
}

File: client/src/app/models/photo.ts
export class Photo {
    title: string;
    description: string;
    photoBase64: string;
    userId: string;
    constructor(title: string, description: string, photoBase64: string, userId: string) {
        this.title = title;
        this.description = description;
        this.photoBase64 = photoBase64;
        this.userId = userId;
    }
}
